commonui/src/test/java/com/hyperwallet/android/ui/common/util/CurrencyParserTest.java

```diff
@@ -105,10 +105,11 @@ public void testGetNumberOfFractionDigits_ThreeDigitDecimal(){
     }
 
     @Test
-    public void testGetCurrency() {
+    public void testCurrencyDetails() {
```

Comment:
we test methods in unit tests and so this should remain `testGetCurrency`

Reply:
Resolved